### ISSUE TITLE
tag a 0.3 and upload tarballs+src RPMs

### ISSUE THREAD

**Issue by mpdehaan** [ansible/ansible#147]
requires pulling tbielawa's spec branch first

**Comment by mpdehaan** [comment]
waiting to merge integration branch (YAML inventory+etc) and that will be 0.0.3

**Comment by mpdehaan** [comment]
this is waiting on the following issues:
- local template support
- built-in facts
- possibly the getvar(host, varname) function -- but not critical

**Comment by andrew-d** [comment]
So, I just ran across this project - cool stuff!  What work have you already done with built-in facts?  I've considered writing a Python version of facter or ohai, and I'd be happy to contribute, if you're interested.  Also, do you think you'll be making it a separate module (a la facter for Puppet, and ohai for Chef), or are you planning on integrating it?

**Comment by mpdehaan** [comment]
Look at the integration branch, built-in facts are part of the setup module.    It's done this way (tailored to Ansible) to avoid any extra software dependencies, so it can still self bootstrap.

Additions to the list of facts included by setup are welcome, maybe discuss on the mailing list?

**Comment by mpdehaan** [comment]
done
